tools/c7n_left/setup.py

```diff
@@ -0,0 +1,66 @@
+# Automatically generated from poetry/pyproject.toml
```

Comment:
is this supposed to be here? @kapilt

Reply:
its odd that its only in this package's setup.py afaics its not in the template we use in tools/dev/poetrypkg.py possibly related to the new poetry version (re 1.3)?

Reply:
by itself it seems harmless, but would be useful to understand the delta